app/helpers/webfinger_helper.rb

```diff
@@ -1,5 +1,16 @@
 # frozen_string_literal: true
 
+# Monkey-patch on monkey-patch.
+# Because it conflicts with the request.rb patch.
+class HTTP::Timeout::PerOperationOriginal < HTTP::Timeout::PerOperation
```

Comment:
I understand why this is needed, but it should be noted that `Timeout.timeout` has worse performance than what we're doing in the `Request` class because it starts a new thread. Perhaps I could change the goldfinger gem to have the same optimizations as the `Request` class...